.azure-pipelines/build-commonlib.yml

```diff
@@ -0,0 +1,24 @@
+pr: none
+trigger: none
+schedules:
+- cron: "0 0 * * *"
+  displayName: Daily build
+  branches:
+    include:
+      - master
+      - 202???
```

Comment:
202??? does not work for yml schedules. But it works for UI configure.
Put it here as a reminder to set it in UI configure.